imports/api/users/user.methods.js

```diff
@@ -16,6 +16,10 @@ Meteor.methods({
     try {
       const userId = Accounts.createUser(values);
       if (userId) {
+        const accountVerification = Accounts.sendVerificationEmail(userId);
+        Meteor.call('sendVerificationLink', accountVerification);
+        // console.log(accountVerification);
+
         Meteor.call('sendWelcomeEmail', userId);
```

Comment:
We should only send the welcome email once user clicks and activates the account...